hibernate-core/src/main/java/org/hibernate/cfg/AvailableSettings.java

```diff
@@ -1725,6 +1725,11 @@
 	 * is enabled logging of Session metrics is enabled by default too.
 	 */
 	String LOG_SESSION_METRICS = "hibernate.session.events.log";
+	
+	/**
+	 * Setting that logs query which executed slower than specified milliseconds. Default is 0 (disabled).
+	 */
+	String LOG_SLOW_QUERY = "hibernate.session.events.log.slow_query";
```

Comment:
The name of this property makes me expect it to be a boolean flag, taking true/false rather than a number.

LOG_QUERIES_SLOWER_THAN_MS ?

Reply:
Fixed in 35a2d98abd80e3ed64734e22f69ec6277b08726a